tests/s3/test_bucket_tagging.py

```diff
@@ -73,29 +74,31 @@ def setup(self):
             assert_utils.assert_true(resp[0], resp)
         self.log.info("ENDED: Test teardown operations.")
 
+    @pytest.fixture(scope="function", autouse=False)
+    def create_bucket(self):
+        """Create bucket for test-cases"""
+        self.log.info("Creating a bucket: %s", self.bucket_name)
+        resp = self.s3_obj.create_bucket(self.bucket_name)
+        assert_utils.assert_true(resp[0], resp[1])
+        assert_utils.assert_equal(resp[1], self.bucket_name, resp[1])
+        self.log.info("Created a bucket: %s ", self.bucket_name)
+
```

Comment:
This is good to have but we can add it in setup fixture as well to avoid one more marker.